pkg/api/bzz.go

```diff
@@ -0,0 +1,99 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package api
+
+import (
+	"bytes"
+	"encoding/binary"
+	"errors"
+	"hash"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"strings"
+
+	bmtlegacy "github.com/ethersphere/bmt/legacy"
+	"github.com/gorilla/mux"
+	"golang.org/x/crypto/sha3"
+
+	"github.com/ethersphere/bee/pkg/jsonhttp"
+	"github.com/ethersphere/bee/pkg/storage"
+	"github.com/ethersphere/bee/pkg/swarm"
+)
+
+func hashFunc() hash.Hash {
+	return sha3.NewLegacyKeccak256()
```

Comment:
I wonder if we could use sha3.New256 not the legacy padding, since we have a new project. This is not in a scope of this PR, but I would like to see if it would be possible.